test/test_utils.py

```diff
@@ -361,6 +361,42 @@ def test_draw_keypoints_colored(colors):
     assert_equal(img, img_cp)
 
 
+@pytest.mark.parametrize("connectivity", [None, [(0, 1)], [(0, 1), (1, 2)]])
+@pytest.mark.parametrize(
+    "vis",
+    [
+        None,
+        torch.ones((2, 3), dtype=torch.bool),
+        torch.ones((2, 3), dtype=torch.int),
+        torch.ones((2, 3, 1), dtype=torch.float),
+    ],
+)
+def test_draw_keypoints_visibility(connectivity, vis):
```

Comment:
Thank you for this test. Do you think we could add something similar to `test_draw_keypoints_vanilla` where we check against a reference image?

The current reference image we're using currently looks like this:

 
![draw_keypoint_vanilla](https://github.com/pytorch/vision/assets/1190450/ea62485e-e63a-4bac-b385-c988e6fe553a)


Instead of creating a new image and add it to the repo, maybe we can just re-purpose this image by just adding new keypoints on the bottom-right corner and marking them as invisible (hopefully that makes sense)?

Reply:
I have done so, but I needed to create a new fake image, because if you look closely, there are two key points near the top left in your image. But one of those should be invisible in our case.